File: app/src/main/java/igrek/songbook/domain/chords/ChordsNotation.java
package igrek.songbook.domain.chords;

import igrek.songbook.R;

public enum ChordsNotation {
	
	GERMAN(1, R.string.notation_german),
	
	ENGLISH(2, R.string.notation_english);
	
	private final int id;
	private final int displayNameResId;
	
	ChordsNotation(int id, int displayNameResId) {
		this.id = id;
		this.displayNameResId = displayNameResId;
	}
	
	public int getId() {
		return id;
	}
	
	public int getDisplayNameResId() {
		return displayNameResId;
	}
}

File: app/src/main/java/igrek/songbook/model/chords/ChordsNotation.kt
package igrek.songbook.model.chords

import igrek.songbook.R

enum class ChordsNotation(val id: Long, val displayNameResId: Int) {

    GERMAN(1, R.string.notation_german),

    ENGLISH(2, R.string.notation_english);

    companion object {
        fun parseById(id: Long): ChordsNotation? {
            return ChordsNotation.values().firstOrNull { v -> v.id == id }
        }
    }
}
